spec/rails_helper.rb

```diff
@@ -86,4 +105,26 @@
 
   config.include Devise::Test::ControllerHelpers, type: :controller
   config.include Warden::Test::Helpers
+
+  config.before do
+    ActiveJob::Base.queue_adapter = :test
+  end
+
+  config.after do
+    # Ensuring we have a clear queue between each spec.
+    ActiveJob::Base.queue_adapter.enqueued_jobs  = []
```

Comment:
There's an ActiveJob::Helper you can call to do this called `clear_enqueued_jobs`

Reply:
Thanks I had in but removed it. I will check it again in a different PR